hack/graph-util.py

```diff
@@ -30,6 +30,10 @@
 _LOGGER = logging.getLogger(__name__)
 _LOGGER.setLevel(logging.DEBUG)
 
+def get_log_version(node):
+    if node['meta']['image-config-data']['architecture']:
```

Comment:
```suggestion
    if node['meta'].get('image-config-data', {}).get('architecture'):
```
so that it would work with images where `image-config-data` is not set

Reply:
Since `meta['image-config-data']['architecture']` is used in other places perhaps a better solution would be creating `def get_architecture(meta)` instead?

Reply:
0f777c8 is using a `get_architecture` wrapper.